index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <div class="nav-bar">
+            <div class="logo">Charles</div>
+            <img src="images/Union.png" alt="Menu Bar">
+        </div>
+        <div class="content">
+            <article>
+                <header>
+                    <h2>I'm Charles Nice to meet you!</h2>
+
+                </header>
+                <p>
+                    My Name is Okoye Charles and I’m a software developer!
+                     I can help you build a product ,
+                    feature or website Look through some of my work and experience! 
+                    If you like what you see and need help on a project, I am available
+                     for hire.
+                </p>
+            </article>
+        
+            <div class="social-header">
+                <p>LET'S CONNECT</p>
+            </div>
+            <nav class="social-links">
+                <ul>
+                    <li><a href="https://twitter.com/OkoyeCharles_"><img src="images/Vector.png" alt="Twitter Icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/charles-okoye-633374236/"><img src="images/Linkedinicon.png" alt="LinkedIn"></a></li>
+                    <li><a href="mailto:okoyecharles509@gmail.com"><img src="images/Gmail.png" alt="Gmail"></a></li>
+                    <li><a href="https://github.com/OkoyeCharles"><img src="images/Github.png" alt="Github"></a></li>
+                    <li><a href="#"><img src="images/Fingers.png" alt="Fingers"></a></li>
```

Comment:
- [ ] your Linkedin icon right now not showing up please try to fix that 
![image](https://user-images.githubusercontent.com/61790587/170371863-122b39bf-3961-41a8-8217-221b533a9ee8.png)

Reply:
done